snapshots/devmapper/snapshotter.go

```diff
@@ -423,15 +422,16 @@ func (s *Snapshotter) createSnapshot(ctx context.Context, kind snapshots.Kind, k
 		}
 		log.G(ctx).Debugf("Creating file system of type: %s with options: %s for thin device %q", s.config.FileSystemType, fsOptions, deviceName)
 		if err := mkfs(ctx, s.config.FileSystemType, fsOptions, dmsetup.GetFullDevicePath(deviceName)); err != nil {
+			errs := []error{err}
 			status, sErr := dmsetup.Status(s.pool.poolName)
 			if sErr != nil {
-				multierror.Append(err, sErr)
+				errs = append(errs, sErr)
```

Comment:
The previous code actually has a bug. If `err` is a `error` type, `multierror.Append(err, sErr)` will create a new `multierror.Error` type and append `sErr` there. It should be `err = multierror.Append(err, sErr)`.